ci-operator/config/codeready-toolchain/host-operator/OWNERS

```diff
@@ -11,6 +11,10 @@ approvers:
 - rajivnathan
 - ranakan19
 - xcoulon
+- fbm3307
+- metlos
+- rsoaresd
+
```

Comment:
let's do these changes in a separate PR please

Reply:
Deal! https://github.com/openshift/release/pull/59764